src/PowerShellEditorServices.Host/CodeLens/CodeLensFeature.cs

```diff
@@ -119,9 +119,8 @@ public CodeLens[] ProvideCodeLenses(ScriptFile scriptFile)
             CodeLensRequest codeLensParams,
             RequestContext<LanguageServer.CodeLens[]> requestContext)
         {
-            var scriptFile =
-                _editorSession.Workspace.GetFile(
-                    codeLensParams.TextDocument.Uri);
+            ScriptFile scriptFile = _editorSession.Workspace.GetFile(
```

Comment:
I like.  The rule for using `var` on my team is this - use `var` when the type of the RHS is obvious i.e. it's a literal, a new statement or type-cast.  And also when required - anonymous types.  Do not use `var` when the type is not obvious, which it usually isn't when invoking a method (or reading a property).

Reply:
100% agreed on that rule!

Reply:
Also 100% agree!